### PR TITLE
Automatically update charts in 2.0 when they change in 1.0 or 1.1

### PR DESCRIPTION
**Pull request by luksa**

### CODE REVIEW

File: prow/config/postsubmits.yaml [hunk 1]
```diff
@@ -418,6 +418,52 @@ postsubmits:
...
+      containers:
+      - image: "quay.io/maistra-dev/maistra-builder:2.0"
+        imagePullPolicy: Always
+        command:
+        - ../test-infra/tools/automator.sh
+        - -o maistra
+        - -r istio-operator
+        - -b maistra-2.0
+        - -f /creds-github/github-token
+        - -c make clean update-1.0-charts update-1.1-charts
```
Comment (jwendell) [inline prow/config/postsubmits.yaml]: can you run `make gen` here and in 2.1 job?

Comment (luksa) [reply prow/config/postsubmits.yaml]: Done.

**Comment by jwendell** [comment]
@luksa you need to run `make gen` and commit the result

File: prow/config/postsubmits.yaml (hunk 1 above)
Comment (luksa) [reply prow/config/postsubmits.yaml]: Do we need `clean`? Probably not, since it's a clean checkout, or isn't it?

Comment (luksa) [reply prow/config/postsubmits.yaml]: Done.

**Comment by jwendell** [comment]
@luksa can you also change 2.1 to run just `make gen`?

**Comment by jwendell** [comment]
I'm also wondering what happens when we:
 merge something in 1.1 : this creates pr's for 2.0 and 2.1
 if the 2.0 pr merges first, will it trigger a 2.1 pr as well?
maybe this is not an issue at all?

**Review by jwendell — APPROVED** [review]

**Comment by luksa** [comment]
I think the 2.0 PR won't result in any changes, so there will be no PR.

**Comment by dgn** [comment]
not sure we still need this, as 2.0.8 is most likely the last 2.0 release of istio-operator

**Comment by jwendell** [comment]
Should we merge or close this one?

**Comment by dgn** [comment]
We should update this to target the maistra-2.1 branch of istio-operator, and include 2.0 chart changes

**Comment by jwendell** [comment]
Closing. 1.0 and 1.1 are EOL.

**Pull request closed without merging**